src/dart-2.md

```diff
@@ -26,6 +26,12 @@ The Dart language, libraries, build system, and web development tools have chang
   * `const` is optional inside of a constant context.
 * Dart no longer has checked mode.
   * [Assert statements][] are still supported, but you enable them differently.
+* Functions marked `async` now run synchronously
+  until the first `await` statement. 
+  * Previously, they would return to the event loop once
+    at the top of the function body before any code runs.
+  * Visit [the Dart newsletter][sync async start] describing
+    the original motivation behind this change.
```

Comment:
We try to mention the reason before telling people to do something that's optional. https://developers.google.com/style/clause-order

```suggestion
  * To learn more, read [the September 2017 Dart newsletter][sync async start].
```